### PR TITLE
Add memcached to dockerfile and docker compose setup

### LINKED ISSUE #1335: Add Memcached to dockerfile and docker compose setup.
Following the addition of Memcached as a cache backend.

### PR DESCRIPTION
**Pull request by emamurho**
Fixes #1335 

- [ ] I added the memcached dependency in docker file and modified the docker compose file accordingly
 I rebuilt the image with the memcache dependency hence the new image name  in docker compose file. 
- [ ] Tests already exists for cache backend
- [ ] Add  name in the contributors file.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,16 +3,19 @@ db:
   environment:
     POSTGRES_USER: postgres
     POSTGRES_PASSWORD: postgres
+cache:
+  image: library/memcached
 web:
-  image: kinto/kinto-server
+  image: ema-kinto-server
```
Comment (Natim) [inline docker-compose.yml]: Oups

File: docker-compose.yml [hunk 2]
```diff
@@ -3,16 +3,19 @@ db:
...
   links:
    - db
+   - cache
   ports:
    - "8888:8888"
   environment:
-    KINTO_CACHE_BACKEND: kinto.core.cache.postgresql
-    KINTO_CACHE_URL: postgres://postgres:postgres@db/postgres
+    KINTO_CACHE_BACKEND: kinto.core.cache.memcached
+    KINTO_CACHE_HOSTS: 127.0.0.1:11211 127.0.0.1:11212
```
Comment (Natim) [inline docker-compose.yml]: Can we use a DNS to the library/memcached image there?

Comment (emamurho) [reply docker-compose.yml]: I made the changes accordingly

File: docker-compose.yml [hunk 3]
```diff
@@ -3,16 +3,19 @@ db:
...
   environment:
-    KINTO_CACHE_BACKEND: kinto.core.cache.postgresql
-    KINTO_CACHE_URL: postgres://postgres:postgres@db/postgres
+    KINTO_CACHE_BACKEND: kinto.core.cache.memcached
+    KINTO_CACHE_HOSTS: cache:11211 cache:11212
     KINTO_STORAGE_BACKEND: kinto.core.storage.postgresql
     KINTO_STORAGE_URL: postgres://postgres:postgres@db/postgres
     KINTO_PERMISSION_BACKEND: kinto.core.permission.postgresql
-    KINTO_PERMISSION_URL: postgres://postgres:postgres@db/postgres
+    KINTO_PERMISSION_URL: postgres://postgres:postgres@db/postgres
```
Comment (Natim) [inline docker-compose.yml]: lot of spaces at the end.

File: kinto/core/__init__.py [hunk 4]
```diff
@@ -28,6 +28,7 @@
     'backoff': None,
     'batch_max_requests': 25,
     'cache_backend': '',
+    'cache_hosts': '',
```
Comment (Natim) [inline kinto/core/__init__.py]: Was this change mandatory to be able to use the KINTO_CACHE_HOSTS env variable? It doesn't work with other backends apparently.

File: kinto/core/storage/postgresql/client.py [hunk 5]
```diff
@@ -81,6 +81,7 @@ def create_from_config(config, prefix='', with_transaction=True):
     settings.pop(prefix + 'max_size_bytes', None)
     settings.pop(prefix + 'prefix', None)
     settings.pop(prefix + 'strict_json', None)
+    settings.pop(prefix + 'hosts', None)
```
Comment (leplatrem) [inline kinto/core/storage/postgresql/client.py]: Now that I see this, maybe a whitelist of supported settings would make more sense...

Comment (emamurho) [reply kinto/core/storage/postgresql/client.py]: Is there something else i need to do?

Comment (leplatrem) [reply kinto/core/storage/postgresql/client.py]: No, don't worry we can open another issue and handle that later :)

File: docker-compose.yml (hunk 3 above)
Comment (leplatrem) [reply docker-compose.yml]: @emamurho please fix this and we're good to go!

@Natim did you try the changes locally?

Comment (Natim) [reply docker-compose.yml]: Yes I did, it seems to works :)

File: kinto/core/storage/postgresql/client.py (hunk 5 above)
Comment (Natim) [reply kinto/core/storage/postgresql/client.py]: I created https://github.com/Kinto/kinto/issues/1410 to handle that.

**Comment by Natim** [comment]
Thanks @emamurho for your work on that!

File: kinto/core/storage/postgresql/client.py (hunk 5 above)
Comment (emamurho) [reply kinto/core/storage/postgresql/client.py]: Hi @Natim, 
Can I also look into this issue (1410) ?

**Comment by leplatrem** [comment]
Thanks @emamurho \o/

File: kinto/core/storage/postgresql/client.py (hunk 5 above)
Comment (Natim) [reply kinto/core/storage/postgresql/client.py]: Yes sure be you guest

**Pull request merged**